director/director.go

```diff
@@ -758,6 +758,16 @@ func registerServeAd(engineCtx context.Context, ctx *gin.Context, sType server_s
 	ctx.Set("serverName", adV2.Name)
 	ctx.Set("serverWebUrl", adV2.WebURL)
 
+	// Iterate over each advertised namespace and join the paths together
+	// into a string where each path is separated by a space
+	// i.e. "<path> <path> <path>"
+	var namespacePaths string
+	for _, namespace := range adV2.Namespaces {
+		path := namespace.Path
+		namespacePaths = fmt.Sprintf("%s %s", namespacePaths, path)
```

Comment:
This will introduce a leading whitespace in your result, can you try trim whitespaces of the final result? `strings.TrimSpace` should be good